### DISCUSSION TITLE
How do I avoid blank first page in my rendered PDF after adding a footer.

### DISCUSSION THREAD

**Discussion by Prav14** [quarto-dev/quarto-cli#8672] [Q&A]
### Description

Hi, I am using typst format in my quarto document to render PDF. When I add footer to the document, the rendered document has a blank first page with just the title and author name. Rest of the contents are moved to the second page of the rendered PDF document. How do I avoid the blank first page and still keep the footer?

Code
````qmd
---
title: "hello-typst"
author: "Test"
format: 
  typst:
    include-in-header: 
        text: |
          #set page( footer: "This is footer")
editor: visual
---

## My first Typst document
````

Quarto version: 1.4

Quarto check output
````qmd
```bash
Quarto 1.4.549
[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.11: OK
      Dart Sass version 1.69.5: OK
      Deno version 1.37.2: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.4.549
      Path: C:\Program Files\Quarto\bin
      CodePage: 1252

[>] Checking tools....................OK
      TinyTeX: (not installed)
      Chromium: (not installed)

[>] Checking LaTeX....................OK
      Tex:  (not detected)

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.2.2
      Path: C:/PROGRA~1/R/R-42~1.2
      LibPaths:
        - C:/Users/MULUKPX/AppData/Local/R/win-library/4.2
        - C:/Program Files/R/R-4.2.2/library
      knitr: 1.42
      rmarkdown: 2.20

[>] Checking Knitr engine render......OK

```
````

Thanks

**Comment by mcanouil** [comment]
Could you properly format your code to be usable? Thank you.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

![A placeholder image](https://placehold.co/600x400.png)

The end.
````
`````

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
